### PR TITLE
T26035 TSC roles

### PR DESCRIPTION
**Pull request by gctucker**
Add descriptions of all the roles handled by the TSC team as previously discussed.  This will need to evolve and be refined, but it should already provide a good enough starting point.

### CODE REVIEW

File: kernelci.org/content/docs/team/tsc.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+* Main repository: [`kernelci-frontend`](https://github.com/kernelci/kernelci-frontend)
+* Ansible config repository: [`kernelci-frontend-config`](https://github.com/kernelci/kernelci-frontend-config)
+* Maintainer: Alexandra
+* Deputy: `gtucker`
+
+### KCIDB
+
+KCIDB provices a set of tools to submit kernel test data to a common database.
+
+* Main repository: [`kcidb`](https://github.com/kernelci/kcidb)
```
Comment (spbnick) [inline kernelci.org/content/docs/team/tsc.md:90]: Can we add [`kcidb-io`](https://github.com/kernelci/kcidb-io/) here for completeness?

File: kernelci.org/content/docs/team/tsc.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+those who have stopped contributing for an extended period of time may be
+removed.  This is done in agreement with the TSC and the AB.
+
+As of today, the committee is composed of the following members:
+
+* [Kevin Hilman](mailto:<khilman@baylibre.com>) - `khilman`
+* [Mark Brown](mailto:<broonie@kernel.org>) - `broonie`
+* [Guillaume Tucker](mailto:<guillaume.tucker@collabora.com>) - `gtucker`
+* [Corentin Labbe](mailto:<clabbe@baylibre.com>) - `montjoie`
+* [Nikolai Kondrashov](mailto:<Nikolai.Kondrashov@redhat.com>) - `spbnick`
```
Comment (spbnick) [inline kernelci.org/content/docs/team/tsc.md:22]: Can you put my personal e-mail here: spbnick@gmail.com instead?

File: kernelci.org/content/docs/team/tsc.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+* [Michał Gałka](mailto:<michal.galka@collabora.com>) - `mgalka`
+* [Alexandra Pereira](mailto:<alexandra.pereira@collabora.com>)
```
Comment (alexandrasp) [inline kernelci.org/content/docs/team/tsc.md:24]: I created a registered user in freenode IRC, please add `apereira` as my nickname :)

File: kernelci.org/content/docs/team/tsc.md (hunk 1 above)
Comment (gctucker) [reply kernelci.org/content/docs/team/tsc.md]: Done.

File: kernelci.org/content/docs/team/tsc.md (hunk 3 above)
Comment (gctucker) [reply kernelci.org/content/docs/team/tsc.md]: Done.

File: kernelci.org/content/docs/team/tsc.md (hunk 2 above)
Comment (gctucker) [reply kernelci.org/content/docs/team/tsc.md]: Done.

**Pull request merged**